client/app/services/file-manager.js

```diff
@@ -59,6 +54,7 @@ export default class FileManager {
   }
 
   uploadFiles() {
+    this.filesToUpload.files.forEach((file) => this.existingFiles.pushObject(file));
```

Comment:
The files won't have an actual link here, though. They don't look like these will be actual "uploaded files"

Reply:
Jotting pseudocode down from our engineering standup:

models/package.js
```
  refreshFileQueue() {
    this.fileManager.existingFiles = this.documents;
  }
```

components/packages/pas-form/edit.js
```
  @action
  async savePackage() {
    await this.args.package.save();
    await this.args.package.reload();
    this.args.package.refreshFileQueue();
  }
```

Reply:
Thanks, I'll try that!

Reply:
The `refreshFileQueue` name was totally wrong... since it's refreshing existing documents. 🙈  I updated it to another possibility. This is all just draft. You have the final call!